### DISCUSSION TITLE
Disable Santized Graphql Query

### DISCUSSION THREAD

**Discussion by alvin562** [open-telemetry/opentelemetry-specification#2716] [General]
Is there an property/flag to disable sanitization of graphql queries within spans?  Something similar to the **otel.instrumentation.common.db-statement-sanitizer.enabled** for databases but for graphql queries.  

Relevant link:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/graphql.md

**Comment by trask** [comment]
hi @alvin562! if you are specifically interested in the Java GraphQL instrumentation, I'd suggest opening discussion in https://github.com/open-telemetry/opentelemetry-java-instrumentation
